game/graphics/renderer.cpp

```diff
@@ -626,6 +711,56 @@ void Renderer::drawFxaa(Encoder<CommandBuffer>& cmd) {
   cmd.draw(Resources::fsqVbo());
   }
 
+void Renderer::applyCmaa2(Tempest::Encoder<Tempest::CommandBuffer>& cmd)
+{
+  uint32_t processCandidatesSetupFlag = 1;
+
+  static bool isFirstRun = true;
```

Comment:
See `gi::fisrtFrame`. 
In fact to handle general case you need not the very first frame, but first, after effect been enabled.